descriptor.md

```diff
@@ -43,6 +43,8 @@ The following fields contain the primary properties that constitute a Descriptor
 
     This OPTIONAL property contains arbitrary metadata for this descriptor.
     This OPTIONAL property MUST use the [annotation rules](annotations.md#rules).
+ 
+Descriptors pointing to [`application/vnd.oci.image.manifest.v1+json`](manifest.md) SHOULD include the extended field `platform`, see [Image Index Property Descriptions](https://github.com/qianzhangxa/image-spec/blob/master/image-index.md#image-index-property-descriptions) for details.
```

Comment:
The link target needs to be `image-index.md#image-index-property-descriptions`.

Also, if this is going to be merged, I think "This property (`platform`) SHOULD be present if its target is platform-specific." in `image-index.md` should be updated to "This property SHOULD be present"